docs/application/dotnet/guides/app-management/cion.md

```diff
@@ -0,0 +1,426 @@
+# Cion
+
+The Cion API provides functionality to communicate with the other devices.
+
+The main features of the `Tizen.Applications.Cion` class include:
+
+- Communicating with other applications in server-client style.
+
+  You can communicate with other applications in [server-client](#server_client) style.
+
+- Communicating with other applications in group style.
+
+  You can communicate with other applications in [group](#group) style.
+
+And you can generate communicate code using [TIDL](#tidl).
+
+## Prerequisites
+
+To enable your application to use the cion functionality:
+
+1.  To use the [Tizen.Applications.Cion](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.Cion.html) class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Comment:
Hello, @sagarkedari 
Please be careful not to guide to use staging link, https://docs1.stg.tizen.org.
This link is staging, which means for the test.

Please guide to use the public link.

Reply:
@mijin85cho , Sorry for providing the test link detail in my comment. I have updated my comment now.

Reply:
Thanks for checking @sagarkedari 
However, would you please follow the reference format? 

With current format, it only goes to API9, even when new TizenFX is released.

**Reference :** https://github.com/Samsung/tizen-docs/blob/52cdb00867da738ebd15afc4c5cce88530cb993b/reviewguide/review_points_guide.md#net-api

 /application/dotnet/api/TizenFX/latest/api/...

Reply:
Per my discussion with @mijin85cho, the link need to be updated as below:
https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.Cion.html

However, since there is some issue with symbolic link "latest" currently, the above suggested link won't be working (OR link is landing on "Page not found" page), until symbolic link gets fixed. The symbolic link issue resolution will be reported to site manager.